tools/mo/openvino/tools/mo/utils/ir_engine/ir_engine.py

```diff
@@ -487,3 +489,13 @@ def __read_old_api_map_element_type(attr, layer_type):
         old_api_map = OldAPIMapElementType(version=version)
         old_api_map.set_legacy_type(element_type)
         return {('old_api_map_element_type', version): old_api_map}
+
+    @staticmethod
+    def __read_old_api_map_common(attr):
```

Comment:
Is it a function to read generic rt_info? If so why is it has old_api in the name?

Reply:
It is mistake, thanks

Reply:
Fixed.